app/templates/events/view/tickets/discount-codes/create.hbs

```diff
@@ -1,10 +1,25 @@
-<div class="ui stackable grid">
-  <div class="row">
-    <h2 class="ui header column">{{t 'Create a new Discount Code'}}</h2>
+{{#if (eq data.ticketsNumber 0)}}
```

Comment:
@kushthedude I pulled your changes to my local and tested. I don't think this fixes the issue.
I created a dummy event with no ticket info but found that the screen was the same as for other event (with tickets).
![Screenshot_2019-05-13 Live Events Open Event](https://user-images.githubusercontent.com/25428397/57588525-27060a80-7533-11e9-89b3-6cad2b7832d9.png)
![Screenshot_2019-05-13 Discount codes Tickets Review1 Events Open Event](https://user-images.githubusercontent.com/25428397/57588527-28373780-7533-11e9-9887-15db2621e31c.png)

Reply:
@shreyanshdwivedi I created the same , But was able to see the screen made by me 
![image](https://user-images.githubusercontent.com/44091822/57588642-b52ec080-7534-11e9-936f-b87e713a0f90.png)
![image](https://user-images.githubusercontent.com/44091822/57588646-b829b100-7534-11e9-9e51-d0a90aeef7c6.png)

Reply:
Oops I think I got your mistake @shreyanshdwivedi to see the changes, 'Click on Create Discount Button'

Reply:
Just a quick question, don't you think that if there is no ticket present then we should show the error on the page attached above itself rather than showing it on clicking `Create` button?
If there is no event ticket then by default there will.be no discount code since beginning (related to the event) and so the table will be empty.
@CosmicCoder96 your views?

Reply:
@shreyanshdwivedi I had this thought before If an Organizer has not provided any ticket info , then there is no need for the Tickets tab under `Explore Event`, but someone told me this behavior was intentional that's why I provided the check after clicking on the button

Reply:
Ah! Here we go again. Check this PR on any event, it is returning same `No Tickets Found` page. @kushthedude

Reply:
@shreyanshdwivedi My wrong accessed the wrong property, Have a look now